### PR TITLE
Daytona Office Hours #9 - Pre-Builds Demo & Latest Fixes!

### PR DESCRIPTION
**Pull request by Amr2812**
## Writer's Checklist

/claim #63

### Writing Structure
- [ x] Use short sentences and paragraphs, and include bucket brigades.
- [ x] Include more than two descriptive H2 headings to organize content.
- [ x] Capitalize headings according to the AP Stylebook ([use this tool](https://capitalizemytitle.com/style/AP/))
- [ x] Include an introduction with at least two paragraphs before the first H2 section.
- [ x] Use appropriate formatting (bold, italic, underline), notes, quotes, TLDRs, and key points.
- [ x] Incorporate `code elements` and `Markdown format` where appropriate.
- [ ] Ensure at least one visual element per “screen” of the article (screenshots, diagrams, tables, graphs, lists, sidenotes, blockquotes).

### Fact-Checking
- [ x] Verify all facts and data points included in the article.

### Asset Management
- [ ] Save images in the `/assets` folder.
- [ x] Follow naming conventions: `YYYYMMDD_title_of_the_article_IMG_NAME_NO.png`.
- [ ] (Optional) Create a GitHub repo for the code referenced in the article and share it.
- [ ] (Optional) Include a link to this Loom video in the PR comments.

### Interlinking
- [ x] Use `CTRL+F` to search for relevant keywords in:
    - [Daytona Dotfiles Sitemap](https://www.daytona.io/sitemap-dotfiles.xml)
    - [Daytona Definitions Sitemap](https://www.daytona.io/sitemap-definitions.xml)
- [ ] Interlink keywords using the appropriate URLs.

### Glossary/Definitions
- [ ] Create new definition in `/defitnitions` folder.

### Review and Edit
- [ x] Ensure articles address the needs of the target audience and their search intent.
- [ x] Read the article out loud to catch any awkward phrasing.
- [ x] Run the draft through [Grammarly](https://grammarly.com) or a similar grammar tool.
- [ x] Double-check adherence to the style guide and repository guidelines.
- [ x] Use the name of the article for the title of the PR.

### CODE REVIEW

File: articles/YYYYMMDD_article_template.md
Comment (nkkko) [inline articles/YYYYMMDD_article_template.md:1]: remove this from commit

File: authors/name_surname.md
Comment (nkkko) [inline authors/name_surname.md:1]: remove this from commit

File: articles/2024822_Daytona_Office_Hours_#9.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Daytona Office Hours #9 - Pre-Builds Demo & Latest Fixes!"
+description: "The recent updates and highlights shared during the Daytona Office Hours #9 and a special demo for Pre-Builds."
+date: 2024-8-22
+author: "Daytona"
+tags: ["Office Hours", "Prebuilds"]
+---
+Daytona-Office-Hours-%239
```
Comment (nkkko) [inline articles/2024822_Daytona_Office_Hours_#9.md:8]: ?

File: articles/2024822_Daytona_Office_Hours_#9.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# Daytona Office Hours #9 - Pre-Builds Demo & Latest Fixes!
+
+## Introduction
+
+In our recent office hours, we discussed the latest development news on Daytona. Even though we haven't had a new release yet, there are some interesting features and fixes we are working on, and looking forward to merging them.
+
+### TL;DR
+
+- **Pre-Builds demo**
```
Comment (nkkko) [inline articles/2024822_Daytona_Office_Hours_#9.md:17]: extend each point just a little bit, see other tldr on the blog

File: articles/2024822_Daytona_Office_Hours_#9.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- **Recent fixes and future releases**
+- **Contributions and bounties**
+
+## Pre-Builds
+
+A couple of weeks back we introduced a major feature called [Project Configs](https://github.com/daytonaio/daytona/pull/789). They allow users to define project configurations in advance and use them when creating workspaces in the future. They contain the relevant repository and build information, environment variables which are then used by default whenever a project with the specified repository is created. They are part of a bigger effort to incorporate [pre-builds](https://github.com/daytonaio/daytona/pull/912) which will let users set up "ready-to-go" builds and spend less time waiting for their development environments to be ready. The documentation for the project config can be found [here](https://www.daytona.io/docs/reference/cli/#daytona-project-config).
```
Comment (nkkko) [inline articles/2024822_Daytona_Office_Hours_#9.md:23]: too big paragraph make it more readable

File: articles/2024822_Daytona_Office_Hours_#9.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+Prebuilds are a way to speed up development by setting a project configuration to listen to changes in the underlying repository and run a build whenever necessary to ensure that once the user wants to create a project and start working, the creation process is much quicker. The user can then decide on a commit interval after which a build should be triggered and any specific trigger files whose changes should immediately start the build process. Prebuilds is still under development, but it is almost ready to be merged. In the office hours video, we presented a demo of creating and managing pre-builds.
```
Comment (nkkko) [inline articles/2024822_Daytona_Office_Hours_#9.md:25]: too big paragraph make it more readable, remove:  In the office hours video, we presented a demo of creating and managing pre-builds.

File: articles/2024822_Daytona_Office_Hours_#9.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+## Recent Fixes
+
+We discussed some of the recent bug fixes:
+- **[Fix .devcontainer.json Detection](https://github.com/daytonaio/daytona/pull/943)**: *Checks if .devcontainer.json is located at the root for auto-detection. It used to check the file at .devcontainer/devcontainer.json only.*
+- **[Handle Binary Download Interruptions](https://github.com/daytonaio/daytona/pull/942)**: *Prefers to use wget instead of curl to download the binary because it automatically handles interruptions. Additionally, add a retry policy if the download fails.*
+- **[Fix Version Mismatch Warning](https://github.com/daytonaio/daytona/pull/941)**: *Fixes the version mismatch warning message.*
+
+Furthermore, there are various minor fixes and optimizations that are going to be in the next release. There are some other open pull requests that we are not sure if they are going to be merged right away.
```
Comment (nkkko) [inline articles/2024822_Daytona_Office_Hours_#9.md:34]: remove if it is no value

File: articles/2024822_Daytona_Office_Hours_#9.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+## Contribute and Win Bounties
+
+We finalized this episode by motivating users to submit issues if they encountered any and to contribute to Daytona open-source to win bounties. If you have any questions, ask us at our [community slack](https://go.daytona.io/slack).
```
Comment (nkkko) [inline articles/2024822_Daytona_Office_Hours_#9.md:38]: this should be a blog article (news) do not reference episode

**Review by nkkko — CHANGES_REQUESTED** [review]

**Comment by Amr2812** [comment]
@nkkko I requested a review last week.

**Comment by nkkko** [comment]
Btw @Amr2812 you should be able to see your badge tomorrow here: 
<p>
  
</p>

**Pull request merged**